src/main/native/glue/a/AaBox.cpp

```diff
@@ -43,6 +44,20 @@ JNIEXPORT jboolean JNICALL Java_com_github_stephengold_joltjni_AaBox_contains
     return result;
 }
 
+/*
+ * Class:     com_github_stephengold_joltjni_AaBox
+ * Method:    contains
+ * Signature: (JJ)Z
+ */
+JNIEXPORT jboolean JNICALL Java_com_github_stephengold_joltjni_AaBox_contains__JJ
+  (JNIEnv *, jclass, jlong boxVal, jlong inOther) {
+    AABox * const pBox = reinterpret_cast<AABox *> (boxVal);
+    AABox * const pOther = reinterpret_cast<AABox *> (inOther);
+
```

Comment:
No blank lines in C++ methods please; they interfere with textual comparison between the .cpp and the .h .